e3nn/o3/_irreps.py

```diff
@@ -423,7 +429,8 @@ def randn(self, *size, normalization='component', requires_grad=False, dtype=Non
         >>> Irreps("5x0e + 10x1o").randn(5, -1, 5, normalization='norm').shape
         torch.Size([5, 35, 5])
 
-        >>> Irreps("2o").randn(2, -1, 3, normalization='norm').norm(dim=1).sub(1).abs().max().item() < 1e-5
+        >>> random_tensor = Irreps("2o").randn(2, -1, 3, normalization='norm')
+        >>> random_tensor.norm(dim=1).sub(1).abs().max().item() < 1e-5
```

Comment:
This rendered a little funny in the docs because the line was too long, so I split it.